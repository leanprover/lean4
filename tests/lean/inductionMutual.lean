mutual
inductive A : Type
| a : A

inductive B : Type
| b : B
end

example (x : PSigma fun (a : A) => True) : A := by
  cases x with | mk x₁ x₂ => ?_
  induction x₁
  done
