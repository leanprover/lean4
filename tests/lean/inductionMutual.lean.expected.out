inductionMutual.lean:11:2-11:14: error: 'induction' tactic does not support mutually inductive types, the eliminator 'A.rec' has multiple motives
